scripts/deployment_dependencies.sh

```diff
@@ -21,6 +21,7 @@ DEPENDENCIES="build-essential \
     libssl-dev \
     libwebsockets-dev \
     libxerces-c-dev \
+    python3-websockets \
```

Comment:
If this is a dependency of a testing or mocking script can we instead add a requirements.txt with this dependency where the script is. This would allow users of the script an easy way to install dependencies of the script if they need it but would prevent us from bloating our software with things that are not actually dependencies of our system